deployer/package.json

```diff
@@ -54,11 +42,7 @@
     "@types/node": "11.13.4",
     "@types/randomstring": "^1.1.6",
     "@types/rimraf": "^3.0.0",
-    "@types/swagger-schema-official": "2.0.15",
```

Comment:
You might need to restore this one, or else fix the root cause for it's being here fixed upstream.

Reply:
Ah - I saw this. Having returned the dep until we fix it upstream.